foodsaving/conversations/emails.py

```diff
@@ -17,76 +17,90 @@
 from foodsaving.webhooks.api import make_local_part
 
 
-def prepare_conversation_message_notification(user, message):
-    if isinstance(message.conversation.target, Group):
-        if message.is_thread_reply():
-            return prepare_group_thread_message_notification(user, message)
-        return prepare_group_conversation_message_notification(user, message)
-    if isinstance(message.conversation.target, PickupDate):
-        return prepare_pickup_conversation_message_notification(user, message)
-    if isinstance(message.conversation.target, GroupApplication):
-        return prepare_group_application_message_notification(user, message)
-    if not message.conversation.target and message.conversation.is_private:
-        return prepare_private_user_conversation_message_notification(user, message)
+def nice_from_email_text(messages):
+    return ', '.join(set(message.author.display_name for message in messages))
+
+
+def prepare_conversation_message_notification(user, messages):
+    assert len(messages) > 0
```

Comment:
`assert` will not be active unless `__debug__` is True - https://docs.python.org/3/reference/simple_stmts.html#assert. I'm not sure what value we would have in production, but could be False.